Vagrantfile

```diff
@@ -7,7 +7,13 @@ settingsFile = ENV["VAGRANT_VARIABLES"] || 'sync/shared/variables.yaml'
 settings = YAML.load_file settingsFile
 k8s_linux_registry=settings['k8s_linux_registry']
 k8s_linux_kubelet_deb=settings['k8s_linux_kubelet_deb']
-k8s_linux_apiserver=settings['k8s_linux_apiserver']
+
+if settings['k8s_linux_apiserver'] == "" then
+    k8s_linux_apiserver="ci/" + `curl https://storage.googleapis.com/kubernetes-release-dev/ci/latest.txt | tr -d '\n'`
```

Comment:
Isn't this the same as using `ci/latest`?

Reply:
@aravindhp nop, please see my above comment when I added ci/latest. (At least in my env)

Reply:
That's interesting because I have been using `ci/latest` without any issues for building from the latest source

Reply:
@aravindhp I am also curios about that. Any caching maybe ? Do you want to try https://github.com/kubernetes-sigs/sig-windows-dev-tools/pull/144 ? :) BE AWARE it will REMOVE all VMS :(

Reply:
>Any caching maybe ? 

Do you mean image caching?

Reply:
yes, somehow the `ci/latest` was using some image available in your server/lab.

Reply:
https://storage.googleapis.com/kubernetes-release-dev/ci/latest.txt - v1.22.6-rc.0.55+487eef699dc8ad
https://storage.googleapis.com/k8s-release-dev/ci/latest.txt - v1.24.0-alpha.1.642+018288b38fcf4c

Reply:
# Cloned fresh repo
```
freshJan17# git clone git@github.com:dougsland/sig-windows-dev-tools.git
Cloning into 'sig-windows-dev-tools'...
remote: Enumerating objects: 1286, done.
remote: Counting objects: 100% (619/619), done.
remote: Compressing objects: 100% (381/381), done.
remote: Total 1286 (delta 325), reused 435 (delta 204), pack-reused 667
Receiving objects: 100% (1286/1286), 90.41 MiB | 11.90 MiB/s, done.
Resolving deltas: 100% (662/662), done.
```

# Change the file to be `ci/latest`:
```
$ vi sync/shared/variables.yaml
```

```
-k8s_linux_apiserver: "ci/v1.22.0-alpha.3.31+a3abd06ad53b2f"   <---- removed
+k8s_linux_apiserver: "ci/latest"   <------ added
```

# Execute make 2
```
$ freshJan17/sig-windows-dev-tools# make 2
echo "cleaning up semaphores..."
cleaning up semaphores...
rm -f up joined cni
echo "######################################"
######################################
echo "Retry vagrant up if the first time the windows node failed"
Retry vagrant up if the first time the windows node failed
echo "Starting the control plane"
Starting the control plane
echo "######################################"
######################################
vagrant up controlplane
cni:
antrea


<snip>
controlplane: [preflight] Some fatal errors occurred:
controlplane: 	[ERROR ImagePull]: failed to pull image gcr.io/k8s-staging-ci-images/kube-apiserver:v1.21.8: output: time="2022-01-17T23:58:25Z" level=fatal msg="pulling image: rpc error: code = NotFound desc = failed to pull and unpack image \"gcr.io/k8s-staging-ci-images/kube-apiserver:v1.21.8\": failed to resolve reference \"gcr.io/k8s-staging-ci-images/kube-apiserver:v1.21.8\": gcr.io/k8s-staging-ci-images/kube-apiserver:v1.21.8: not found"
<snip>
```